lib/parse.js

```diff
@@ -10,6 +10,12 @@ var DOMParser = require('xmldom').DOMParser;
 
 exports.parse = parse;
 
+var NODE_TYPES = {
```

Comment:
I don't think I have access to sauce, nor am I familiar with the service in great detail. @TooTallNate care to take this on?